### PR TITLE
enable newrelic apm for new django workers

### PR DESCRIPTION
**Pull request by snopoke**
@dannyroberts

### CODE REVIEW

File: fab/fabfile.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def format_env(current_env, extra=None):
     host = current_env.get('host_string')
     if host in current_env.get('new_relic_enabled', []):
         ret['new_relic_command'] = '%(virtualenv_root)s/bin/newrelic-admin run-program ' % env
-        ret['supervisor_env_vars'] = 'NEW_RELIC_CONFIG_FILE=../newrelic.ini,NEW_RELIC_ENVIRONMENT=%(environment)s' % env
+        ret['supervisor_env_vars'] = 'NEW_RELIC_CONFIG_FILE="%(root)s/newrelic.ini",NEW_RELIC_ENVIRONMENT="%(environment)s"' % env
```
Comment (dimagimon) [inline fab/fabfile.py]: E501 line too long (130 > 115 characters)

**Comment by czue** [comment]
did you already test this on prod? :+1: from me if we're confident this won't bring down HQ again. otherwise i might propose testing on one worker first

**Comment by TylerSheffels** [comment]
Is there a way to tell supervisor to start all of the configurations that are valid, rather than not starting any when one is broken?  Seems like that would have caused at least django to come back up in this instance.

**Comment by snopoke** [comment]
I've teststed the format on hqdjango6 and tested locally that running the
command produces the right format but I haven't tested deploying to a
single node since that requires a code update. What I'll do is split this
up to make the fabfile changes and not the env changes which will make it
easy to test once the fabfile changes get deployed.

**Comment by snopoke** [comment]
Updated with just the fabfile changes. This should be a noop change as far as current deploy is concerned.

**Comment by TylerSheffels** [comment]
Sounds good to me.

**Pull request merged**